test/docker-compose.yml

```
@@ -6,20 +6,6 @@ services:
     container_name: activemq-classic
```

Hint: if you add `init: true` the shutdown should work as expected and we can get rid of the other `activemq-dev` container.

Also I would prefer if we pin the version of the container image, otherwise the image will get updated and our tests might break over time, leaving us wondering why it suddenly stopped working.
```suggestion
    image: apache/activemq-classic:6.0.1
    container_name: activemq-classic
    init: true
```

---

Yup, I'll go and pin the other connector image versions aswell.